Sources/UIKitBackend/UIKitBackend+Control.swift

```diff
@@ -332,7 +347,7 @@ extension UIKitBackend {
     ) {
         let textEditorWidget = textEditor as! TextEditorWidget
 
-        textEditorWidget.child.isEditable = environment.isEnabled
```

Comment:
Would we have to still set `.child.isEditable` somewhere? This change seems like it'd break something.

Reply:
So it turns out that (on iOS) deactivating the text editor in this way still lets you navigate to it, it's just that typing does nothing once you do. Setting `isEditable` doesn't let you navigate to it. I've updated the `didSet` accordingly, leaving the behavior I had for tvOS and using `child.isEditable` on other systems.